service/src/main/java/bio/terra/pipelines/db/entities/PipelineRun.java

```diff
@@ -46,8 +46,11 @@ public class PipelineRun {
   @Column(name = "workspace_name")
   private String workspaceName;
 
-  @Column(name = "workspace_storage_container_url", nullable = false)
-  private String workspaceStorageContainerUrl;
+  @Column(name = "workspace_storage_container_name", nullable = false)
+  private String workspaceStorageContainerName;
+
+  @Column(name = "workspace_google_project", nullable = false)
```

Comment:
whats the difference between `workspace_google_project` and `workspace_project`?

Reply:
chatted with faces, think we can rename `workspace_project` to `workspace_billing_project` or `workspace_terra_project` to be more clear